src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs

```diff
@@ -33,9 +36,17 @@ private SearchViewModel ViewModel
             get { return DataContext as SearchViewModel; }
         }
 
+        private WorkspaceView WorkspaceView;
```

Comment:
Hi @aosyatnik, minor thing but private fields should have names such as `workspaceView`.